src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2973,6 +2976,102 @@ Activate by holding the key for the character you want to add an accent to, then
+  </data>
+  <data name="FancyZones_PreviousLayout" xml:space="preserve">
+    <value>Previous Layout</value>
+  </data>
+  <data name="FancyZones_NextLayout" xml:space="preserve">
+    <value>Next Layout</value>
+  </data>
+  <data name="FancyZones_Launch" xml:space="preserve">
+    <value>Launch</value>
+  </data>
+  <data name="FileLocksmith_ShortDescription" xml:space="preserve">
+    <value>Right-click on files or directories to show running processes</value>
+  </data>
+  <data name="FindMyMouse_ShortDescription" xml:space="preserve">
+    <value>Finds the mouse</value>
+  </data>
+  <data name="Hosts_Launch" xml:space="preserve">
+    <value>Launch</value>
+  </data>
+  <data name="ImageResizer_ShortDescription" xml:space="preserve">
+    <value>Select Image Resizer in the right-click context menu</value>
+  </data>
+  <data name="KeyboardManager_RemapAKey" xml:space="preserve">
+    <value>Remap a key</value>
+  </data>
+  <data name="KeyboardManager_RemapAShortcut" xml:space="preserve">
+    <value>Remap a shortcut</value>
+  </data>
+  <data name="MouseHighlighter_ShortDescription" xml:space="preserve">
+    <value>Highlights clicks</value>
+  </data>
+  <data name="MouseJump_ShortDescription" xml:space="preserve">
+    <value>Quickly move the mouse pointer</value>
+  </data>
+  <data name="MouseCrosshairs_ShortDescription" xml:space="preserve">
+    <value>Draw crosshairs centered on the mouse pointer</value>
+  </data>
+  <data name="MouseWithoutBorders_ShortDescription" xml:space="preserve">
+    <value>Move your cursor across multiple devices</value>
+  </data>
+  <data name="PastePlain_ShortDescription" xml:space="preserve">
+    <value>Paste clipboard content without formatting</value>
+  </data>
+  <data name="Peek_ShortDescription" xml:space="preserve">
+    <value>Quick and easy previewer</value>
+  </data>
+  <data name="PowerRename_ShortDescription" xml:space="preserve">
+    <value>Select Power Rename in right-click context menu</value>
+  </data>
+  <data name="Run_ShortDescription" xml:space="preserve">
+    <value>A quick launcher</value>
+  </data>
+  <data name="PowerAccent_ShortDescription" xml:space="preserve">
+    <value>An alternative way to type accented characters</value>
+  </data>
+  <data name="RegistryPreview_ShortDescription" xml:space="preserve">
+    <value>Visualize and edit Windows Registry files</value>
+  </data>
+  <data name="ScreenRuler_ShortDescription" xml:space="preserve">
+    <value>Measure pixels on your screen</value>
+  </data>
+  <data name="ShortcutGuide_ShortDescription" xml:space="preserve">
+    <value>Shows a help overlay with Windows shortcuts</value>
+  </data>
+  <data name="PowerOcr_ShortDescription" xml:space="preserve">
+    <value>Shows a help overlay with Windows shortcuts</value>
+  </data>
+  <data name="Dashboard_Activation" xml:space="preserve">
+    <value>Activation</value>
+  </data>
+  <data name="RegistryPreview_Launch" xml:space="preserve">
+    <value>Launch</value>
```

Comment:
```suggestion
    <value>Open</value>
```

Reply:
This resource has been deleted, this one is used:
 <data name="Launch_RegistryPreview.Content" xml:space="preserve">
   <value>Launch Registry Preview</value>

Reply:
> Don't use to mean opening an app or form. Use _open_ instead.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/launch

Reply:
Thanks for the comment. We have decided that we keep the "launch" phrases for now, for this PR and discuss later what should be changed according to the guideline and do the update in a separate PR.